<ui version="4.0" >
 <class>Programa</class>
 <widget class="QDialog" name="Programa" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Programa [%1]</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Nome: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nomeEdit" >
       <property name="maxLength" >
        <number>128</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="textEdit" />
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="importButton" >
       <property name="text" >
        <string>&amp;Impotar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton" >
       <property name="text" >
        <string>E&amp;xportar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="submitButton" >
       <property name="text" >
        <string>&amp;Gravar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>&amp;Fechar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
